merge master into live-based

luyajun0205 opened the pull request.

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Merge branch 'live-based' into master

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

v-susu-zz commented:
Docs Build status updates of commit _[cec2dc4](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/pull/60/commits/cec2dc4c7d04efc6de9cfd0cb2420f139d2e4b92)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/vm/image?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app/credential.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app/credential.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app/credential?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app/owner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app/owner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app/owner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app/permission.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app/permission.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app/permission?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group/member.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group/member.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group/member?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group/owner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group/owner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group/owner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/signed-in-user.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/signed-in-user.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/signed-in-user?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp/credential.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp/credential.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp/credential?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp/owner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp/owner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp/owner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/user.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/user.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/user?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ext/aks-preview/aks.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ext/aks-preview/aks.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ext/aks-preview/aks?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/address-pool.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/address-pool.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/address-pool?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/auth-cert.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/auth-cert.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/auth-cert?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/frontend-ip.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/frontend-ip.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/frontend-ip?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/frontend-port.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/frontend-port.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/frontend-port?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/http-listener.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/http-listener.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/http-listener?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/http-settings.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/http-settings.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/http-settings?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/probe.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/probe.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/probe?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/rewrite-rule.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/rewrite-rule.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/rewrite-rule?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/rewrite-rule/set.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/rewrite-rule/set.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/rewrite-rule/set?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/rule.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/rule.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/rule?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/ssl-cert.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/ssl-cert.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/ssl-cert?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/ssl-policy.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/ssl-policy.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/ssl-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/url-path-map.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/url-path-map.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/url-path-map?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/url-path-map/rule.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/url-path-map/rule.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/url-path-map/rule?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/application-gateway/waf-config.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/application-gateway/waf-config.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/application-gateway/waf-config?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/a.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/a.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/a?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/aaaa.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/aaaa.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/aaaa?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/cname.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/cname.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/cname?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/mx.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/mx.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/mx?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/ns.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/ns.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/ns?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/ptr.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/ptr.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/ptr?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/soa.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/soa.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/soa?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/srv.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/srv.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/srv?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/record-set/txt.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/record-set/txt.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/record-set/txt?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/dns/zone.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/dns/zone.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/dns/zone?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/express-route.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/express-route.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/express-route?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/express-route/gateway.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/express-route/gateway.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/express-route/gateway?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/express-route/gateway/connection.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/express-route/gateway/connection.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/express-route/gateway/connection?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/express-route/peering.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/express-route/peering.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/express-route/peering?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/network/express-route/peering/connection.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/network/express-route/peering/connection.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/express-route/peering/connection?view=azure-cli-2017-03-09-profile&branch=pr-en-us-60) (azure-cli-2017-03-09-profile) |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C27%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201905270613585191-60%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pSb1d8Cerhok5pvRT8mIDhtSu6DeyAMWC1jOLNlXYeg%3D&st=2019-05-27T06%3A19%3A06Z&se=2019-06-27T06%3A24%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Add proxy information to install troubleshooting.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

sptramer committed:
Merge pull request #1471 from sptramer/troubleshooting/proxy

Add proxy information to install troubleshooting.

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=538907845
https://github.com/Azure/azure-cli-extensions/commit/f35293575fb339a4dec8c78d662f4b0fbe5fb005

sptramer committed:
Merge pull request #1473 from MicrosoftDocs/az-ext-list-ci-538907845

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Fix issues from 190529 github triage [Closes #1469]

sptramer committed:
Merge pull request #1474 from sptramer/triage-fixes-190529

Fix issues from 190529 github triage [Closes #1469]

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

sptramer committed:
Add displayName search terms to ToC (#1479)

* Add displayName search terms to ToC

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=539851428
https://github.com/Azure/azure-cli-extensions/commit/5dd870a2e4dc0115c6763f875853cd7d772a914f

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

sptramer committed:
Merge pull request #1480 from MicrosoftDocs/az-ext-list-ci-539851428

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Azure CLI 2.0.66 release notes.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=540908562
https://github.com/Azure/azure-cli-extensions/commit/93e62839d39293ac716cff4d810d8bc46c36f980

sptramer committed:
Merge pull request #1482 from MicrosoftDocs/az-ext-list-ci-540908562

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=541395206
https://github.com/Azure/azure-cli-extensions/commit/3110cfbb4a417a69e957e5366ba89d9ee269fa6c

sptramer committed:
Merge pull request #1483 from MicrosoftDocs/az-ext-list-ci-541395206

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Copyedit release notes.

sptramer committed:
Merge pull request #1481 from adewaleo/cli-release-notes-2.0.66

Azure CLI 2.0.66 release notes.

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=543215250
https://github.com/Azure/azure-cli-extensions/commit/5422ae88252a93dabd30959af6c39749f765e4b5

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

sptramer committed:
Merge pull request #1488 from MicrosoftDocs/az-ext-list-ci-543215250

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=544388890
https://github.com/Azure/azure-cli-extensions/commit/fde1d34b90fbf66b280a76c076e59846f16b9b8f

sptramer committed:
Merge pull request #1490 from MicrosoftDocs/az-ext-list-ci-544388890

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)
